casepro/statistics/signals.py

```diff
@@ -38,6 +41,29 @@ def record_new_outgoing(sender, instance, created, **kwargs):
         if instance.partner:
             DailyCount.record_item(day, DailyCount.TYPE_REPLIES, partner)
 
+        if case:
+            # count the very first response on an org level
+            if instance == case.outgoing_messages.earliest('created_on'):
```

Comment:
The first scenario is definitely one that caters for our case. 
The second scenario does warrant consideration, but I don't think looking at averages alone will solve the oversight problems. If a partner reports a very low average for initial response, but then has a horrible case-closing record, that would show a potential abuse of the system by immediately responding. But it might also just speak to the different workflows. If partners always respond to cases with "Thank you for your message. We're looking into it" then the _average time to respond_ becomes a useless indicator.

I'm happy to add the second scenario to our TODO list, but I don't see it as a blocker at the moment. Also, I've been having some conversations with our team for looking at different ways of visualising this information, because the current table is not really ideal for this. You'd much rather have a graph that shows historical data than just a "Last month/This month" number. We'll add this to our TODO list as well.

We can definitely spend more time hashing out different use cases and potential abuses, but we